### PR TITLE
57 add legend for module status

### PR DESCRIPTION
**Pull request by noobiba**
@lauralindal @sarahkami Please review for merge. Thank you! :)

### CODE REVIEW

**Review by sarahkami — APPROVED** [review]
I would move the static html from the react component to the html file. But if needed we can do it later on. So I approve ;)

File: src/ModulePlan.js [hunk 1]
```diff
@@ -5,19 +5,52 @@ class ModulePlan extends React.Component {
...
-            )
-          })
-          }
-        </div>
+
+        <legend>
+          <div className="flex-container">
+            <div className="flex-item item1">
+              <div className="available"></div>
+              <div><h6>Kurs belegt</h6></div>
```
Comment (lauralindal) [inline src/ModulePlan.js]: "Kurs belegbar" or "Kurs verfügbar" would be better fitting, please adjust that :)

File: src/ModulePlan.js [hunk 2]
```diff
@@ -5,19 +5,52 @@ class ModulePlan extends React.Component {
...
+              <div className="urgent"></div>
+              <div><h6>Kurs mit Dringlichkeit</h6></div>
+            </div>
+            <div className="flex-item item5">
+              <div className="selected"></div>
+              <div><h6>Kurs geplant</h6></div>
+            </div>
+            <div className="flex-item item6">
+              <div className="selected_urgent"></div>
+              <div><h6>Dringlichkeitskurs geplant</h6></div>
```
Comment (lauralindal) [inline src/ModulePlan.js]: "Dringlicher Kurs geplant", maybe?

File: src/ModulePlan.js [hunk 3]
```diff
@@ -5,19 +5,52 @@ class ModulePlan extends React.Component {
...
+              <div className="available"></div>
+              <div><h6>Kurs belegt</h6></div>
+            </div>
+            <div className="flex-item item2">
+              <div className="started"></div>
+              <div><h6>Kurs begonnen</h6></div>
+            </div>
+            <div className="flex-item item3">
+              <div className="completed"></div>
+              <div><h6>Kurs beendet</h6></div>
```
Comment (lauralindal) [inline src/ModulePlan.js]: "Kurs abgeschlossen" would be more accurate :)

**Review by lauralindal — CHANGES_REQUESTED** [review]

**Review by lauralindal — APPROVED** [review]

**Pull request merged**